pytest_factoryboy/fixture.py

```diff
@@ -16,16 +16,15 @@
 
 from .codegen import FixtureDef, make_fixture_model_module
 from .compat import PostGenerationContext
-from typing import TYPE_CHECKING, overload, cast
-from typing_extensions import Protocol, TypeAlias
 
 if TYPE_CHECKING:
     from typing import Any, Callable, TypeVar
-    from _pytest.fixtures import SubRequest, FixtureFunction
+
+    from _pytest.fixtures import FixtureFunction, FixtureRequest, SubRequest
```

Comment:
FixtureFunction doesn't exist

Reply:
I have tested it on `pytest==7.1.2` and it exists, it's defined [here](https://github.com/pytest-dev/pytest/blob/main/src/_pytest/fixtures.py#L81).

Reply:
But I have checked other versions and it seems that type hintings for `pytest` were added in the `6.2.0` version. References:

+ 3 point of [6.2.0 release notes](https://docs.pytest.org/en/6.2.x/changelog.html#improvements)
+ [GitHub issue](https://github.com/pytest-dev/pytest/issues/7469)

Taking this into account I think we should drop support for `pytest<6.2.0` because maintaining type hintings will require more effort and potentially slow down the development of new features.

Reply:
Or at least let's drop `pytest<6.0` because it seems like types were initially added in this release - [commit adding type hintings](https://github.com/pytest-dev/pytest/commit/28338846884687cf57c4add215a84c3c75085ac1)

Reply:
I'm not sure how this will affect developers using `pytest<6.2.0` and type hintings, will they be able to normally use `pytest-factoryboy`?

Reply:
I think so, I think that running mypy against a project that has pytest-factoryboy as a dependency just uses our types, it doesn't check if they are actually correct. Users will run mypy against their source files, not against pytest-factoryboy.